main.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+require_once __DIR__ . '/vendor/autoload.php';
+
+    $instaobj = Instamojo\Instamojo::init('app',[
+        "client_id" =>  $_ENV["CLIENT_ID"],
+        "client_secret" => $_ENV["CLIENT_SECRET"]
+       
+    ],true);
+ 
+    
+    // $transaction_id = "TEST_".time();
```

Comment:
Lets not leave commented code in the commits. Delete if this is not required in production